py3status/modules/deadbeef.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Display track currently playing in deadbeef.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh usage in seconds (default: 1s)
+    - format : see placeholders below
+    - delimiter : delimiter character for parsing (default: ¥)
+
+Format of status string placeholders:
+    {artist} : artist
+    {title} : title
+    {elapsed} : elapsed time
+    {length} : total length
+    {year} : year
+    {tracknum} : track number
+
+@author mrt-prodz
+"""
+from subprocess import check_output
+from time import time
+
+
+class Py3status:
+    # available configuration parameters
+    cache_timeout = 1
+    delimiter = '¥'
```

Comment:
Will fail on `Maximum the Hormone — Black ¥ Power G-MEN Spy` etc, We could use some long random string instead, e.g. `ooxohF2uDief8DooAhcaixu3Ju7doo2i`